src/modules/components/asset/assetDataListItem/assetDataListItemStructure/assetDataListItemStructure.tsx

```diff
@@ -0,0 +1,105 @@
+import classNames from 'classnames';
+import type React from 'react';
+import { Avatar, DataList, NumberFormat, Tag, formatterUtils, type IDataListItemProps } from '../../../../../core';
+
+export interface IAssetDataListItemStructureProps extends IDataListItemProps {
+    /**
+     * The source of the logo for the token.
+     */
+    logoSrc?: string;
+    /**
+     * The name of the Token.
+     */
+    tokenName?: string;
+    /**
+     * The symbol of the Token.
+     */
+    symbol?: string;
+    /**
+     * The amount of the Token.
+     */
+    amount?: number | string;
+    /**
+     * The price of the Token.
+     */
+    USDAmount?: number | string;
+    /**
+     * changed price amount (E.g. in last 24h).
+     */
+    changedAmount?: number | string;
+    /**
+     * changed price amount ratio (E.g. in last 24h).
+     */
+    changedPercentage?: number | string;
+}
+
+export const AssetDataListItemStructure: React.FC<IAssetDataListItemStructureProps> = (props) => {
+    const { logoSrc, tokenName, amount, symbol, USDAmount, changedAmount, changedPercentage, ...otherProps } = props;
+
+    const formattedChangedAmount = Number(Number(changedAmount).toFixed(2));
+    const formattedChangedPercentage = Number(Number(changedPercentage).toFixed(2));
+
+    const sign = (value: number) => (value > 0 ? '+' : value < 0 ? '-' : '');
```

Comment:
Btw there is a Math.sign() function in Javascript that returns 1, 0 & -1 based on the given value.

Reply:
Yeah, but it doesn't return the sign. for example for positive it removes + sign but I want to keep it. Essentially, it is not as helpful as it might seem